docs/backup_workstations.rst

```diff
@@ -118,16 +118,92 @@ Then, copy the contents of the device's persistent volume to the directory using
 
 .. code:: sh
 
-  rsync -a --info=progress2 --no-specials --no-devices \
-      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup
+  sudo bash -c "rsync -a --info=progress2 --no-specials --no-devices \
+      /media/amnesia/TailsData/ /media/amnesia/Backup/admin-backup && sync"
 
-Once complete, unmount the TailsData partition.
+.. note:: Note that the first path argument ends with a ``/``, but the second 
+          path argument should not!
```

Comment:
See comment below regarding trailing slashes and rsync; same nit applies here.

Reply:
Added similar abbreviated version of the above.